docs/sources/writing-guide/contribute-documentation/index.md

```diff
@@ -121,7 +144,7 @@ Depending on the size of the PR and the priority of other work, the PR will eith
 When you edit the `main` branch of a project, it affects the content in the `next` directory of the website.
 To edit a previous version, or `latest` (the most recent release), you must backport the changes into the long-lived version branches in the project repository.
 
-To backport a change, use the `backport <vMAJOR.MINOR.x>` labels on the GitHub pull request.  For more information, see [backporting]({{< relref "../tooling-and-workflows/backporting/" >}}).
+To backport a change, use the `backport <vMAJOR.MINOR.x>` labels on the GitHub pull request.
```

Comment:
Why did you remove the link that points out that we have an entire other topic about backporting?  (I'm seeing you remove a lot of updates that I have recently inserted because I thought they were needed...)

Reply:
Again, I didn't do so intentionally. Are you sure your additions were merged into the live repo?

Reply:
I know, it seems strange to even ask since we can see them. But I didn't delete these.